src/client/results/mostfrequent/users/UserItemsList.js

```diff
@@ -14,6 +14,7 @@ class UserItemsList extends Component {
     $('.ui.dropdown.userwords').dropdown({
       action: 'nothing',
     });
+    $('.userItemsList').css('height', `${$(window).height() - ($(window).height() / 5)}px`);
```

Comment:
What's wrong with height: 80%? I guess though you need to have set height: 100% on every element above it for that to work which we've not done yet